Adafruit_IO/client.py

```diff
@@ -231,17 +243,52 @@ def receive_previous(self, feed):
         path = "feeds/{0}/data/previous".format(feed)
         return Data.from_dict(self._get(path))
 
-    def data(self, feed, data_id=None):
+    def data(self, feed, data_id=None, max_results=API_PAGE_LIMIT):
```

Comment:
Does a call to this function return the same value as the `master` branch?

Reply:
Yes, this is backwards compatible. If max_results isn't provided (legacy code wouldn't provide it), up to 1000 points will be returned, as before.